lib/inventory_sync/async/inventory_scheduled_sync.rb

```diff
@@ -24,6 +30,11 @@ def plan_org_sync(org)
         plan_action InventoryFullSync, org
       end
 
+      def plan_remove_insights_hosts
+        # plan a remove hosts action with search set to empty (all records)
```

Comment:
I am assuming this is empty because we will fill that in with the string from the UI or API?

Reply:
This is the scheduled automatic job, it will remove everything, that's why the search clause is empty here. In the manual process, we will need to pass the search term from the client side.